src/js/pensions/components/GetFormHelp.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function GetFormHelp() {
```

Comment:
Thanks @jbalboni, have moved the shared component to common/schemaform folder and renamed as `GetPensionOrBurialFormHelp`, please let me know if another name or location for this would be better.